pkgs/applications/graphics/makeicns/patch-IconFamily.m.diff

```diff
@@ -0,0 +1,57 @@
+--- makeicns/IconFamily.m	2012-01-19 22:16:19.000000000 +0100
++++ IconFamily.m	2013-10-29 15:51:30.000000000 +0100
+@@ -85,7 +85,9 @@
+ 
+ + (Handle) get32BitDataFromBitmapImageRep:(NSBitmapImageRep*)bitmapImageRep requiredPixelSize:(int)requiredPixelSize;
+ 
++#if MAC_OS_X_VERSION_MIN_REQUIRED < 1090
```

Comment:
10.12 currently.

Reply:
It looks like this patch removes these functions as they've been deprecated on later versions of MacOS.  I've changed the patch so it explicitly removes this code rather than using the preprocessor to remove it.